src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.String.cs

```diff
@@ -22,21 +22,23 @@ public byte[] ReadByteString()
             return result;
         }
 
-        public int ReadByteString(Span<byte> buffer)
+        public bool TryReadByteString(Span<byte> buffer, out int bytesWritten)
```

Comment:
```suggestion
        public bool TryReadByteString(Span<byte> destination, out int bytesWritten)
```

(Buffer-writing methods should use the name destination unless it's ambiguous or there's another more powerful naming convention, like matching something an overload did in a previous release)